### PR TITLE
Create Bolu

### PR DESCRIPTION
**Pull request by Enescaffrey**
Bolu sabe tez

### CODE REVIEW

File: Bolu [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+<id>https://csl.mendeley.com/styles/596285991/Bolusabe</id>
+<link href="http://www.zotero.org/styles/brachytherapy" rel="self"/>
+<link href="http://www.zotero.org/styles/the-journals-of-gerontology-series-a" rel="template"/>
+<link href="https://www.elsevier.com/journals/brachytherapy/1538-4721/guide-for-authors" rel="documentation"/>
+<author>
+<name>Patrick O'Brien</name>
+</author>
+<contributor>
+<name>Enes Caffrey</name>
+<uri>https://www.mendeley.com/profiles/enes-caffrey/</uri>
```
Comment (POBrien333) [inline Bolu:13]: Make yourself the author and then remove yourself as contributor.

File: Bolu [hunk 2]
```diff
@@ -0,0 +1,238 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="expanded">
+<info>
+<title>Brachytherapy - Enes Caffrey</title>
```
Comment (POBrien333) [inline Bolu:3]: Remove your name from title.

**Comment by POBrien333** [comment]
Can you make your edits, then run the code through this tool to get the correct indentation again, please?
formatter.citationstyles.org/

**Comment by rmzelle** [comment]
@Enescaffrey, thanks for your submission. Are you trying to make corrections to the existing Brachytherapy CSL style, or did you modify it for something else?

Google Translate is failing me on "Bolu sabe tez" so I don't understand your description.

**Comment by rmzelle** [comment]
@Enescaffrey, if you write back to us we can always reopen this pull request.

**Pull request closed without merging**